types/office-js-preview/index.d.ts

```diff
@@ -17200,6 +17393,122 @@ declare namespace Office {
         removeHandlerAsync(eventType: Office.EventType, callback?: (asyncResult: Office.AsyncResult<void>) => void): void;
     }
 
+    /**
+     * Represents the categories on an item.
+     *
+     * [Api set: Mailbox Preview]
+     *
+     * @remarks
+     * <table>
+     *   <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions | Minimum permission level}</td><td>ReadItem</td></tr>
+     *   <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/#extension-points | Applicable Outlook mode}</td><td>Compose or Read</td></tr>
+     * </table>
+     * 
+     * @beta
+     */
+    interface Categories {
+        /**
+         * Adds categories to an item. Each category name must be unique to that mailbox but multiple categories can use the same color.
```

Comment:
We need to document that the categories being added need to be present in the master category list else the API will fail.

Reply:
Added